CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [9.1.0] - Unreleased
+
+# Adds
+
+-   Adds UserMetadata recipe
```

Comment:
Need to discuss:
- By default, should we tie this recipe to other recipes? For example, other auth recipes do an init of user metadata recipe in them, and when fetching user info, they also fetch the user's metadata info and return it to the user.
- In emailpassword recipe, we allow custom form fields, should that be saved in metadata automatically by default?
- How will this be tied to Session grants? And should we do something about this now?

Reply:
Ok, let's discuss this tomorrow, but my feelings so far:
- I don't think so, I think it should be a separate recipe. 
  - If we want to tie everything into the auth recipes their config will bloat and we'd also have to do it on the frontend which would increase the bundle size.
  - I don't think we should automatically fetch this info. We shouldn't control how/when they are refreshed and if (or how) they are cached.
- We've discussed this in the progressive profiling part of the session grants docs, but I don't think we need to do anything for now. When session grants are implemented we export a grant in this recipe for requiring/validating fields in the metadata but I think that's about it.

Reply:
I agree with everything, except for adding a new config to the ep recipe. So resolving this